src/Tizen.TV.Extension.UIControls.Forms/ITVESPlayer.cs

```diff
@@ -0,0 +1,162 @@
+        /// <summary>
+        /// The matroska color information. This value is only for video packet. If you set this value on a packet of other type, you can see an error when you submit the packet.
+        /// </summary>
+        public ESMatroskaColor? matroskaColorInfo { get; set; }
+    }
+
+    /// <summary>
+    /// Represents one of es packet which includes handle of data inside the trust zone,
+    /// contains stream type, handle for encrypted es data packet, handle size, pts and duration.
+    /// </summary>
+    public struct ESHandlePacket
+    {
+        /// <summary>
+        /// Stream type of ESPacket
+        /// </summary>
+        public StreamType type;
+        /// <summary>
+        /// Handle for encrypted es packet inside the trust zone
+        /// </summary>
+        public uint handle;
+        /// <summary>
+        /// Handle size for the handle of ESHandlePacket
+        /// </summary>
+        public uint handleSize;
+        /// <summary>
+        /// PTS(Presentation Time Stamp) of es packet
+        /// </summary>
+        public ulong pts;
+        /// <summary>
+        /// DUration of es packet
+        /// </summary>
+        public ulong duration;
+        /// <summary>
+        /// The matroska color information. this value is only for video packet. If you set
+        /// this value on a packet of other type, you can see an error when you submit the
+        /// packet.
+        /// </summary>
+        public ESMatroskaColor? matroskaColorInfo;
+    }
+
+
+    /// <summary>
+    /// Represents stream information for video stream which is demuxed from application,
+    /// contains codec data, mime type, width, height, max width, max hgeight and frame
+    /// rate(num, den).
+    /// </summary>
+    public class VideoStreamInfo
+    {
+        /// <summary>
+        /// Codec data for the associated video stream
+        /// </summary>
+        public byte[] codecData { get; set; }
+        /// <summary>
+        /// Mime type for the associated video stream
+        /// </summary>
+        public VideoMimeType mimeType { get; set; }
+        /// <summary>
+        /// width for the associated video stream
+        /// </summary>
+        public int width { get; set; }
+        /// <summary>
+        /// Height for the associated video stream
+        /// </summary>
+        public int height { get; set; }
+        /// <summary>
+        /// Max width for the associated video stream
+        /// </summary>
+        public int maxWidth { get; set; }
+        /// <summary>
+        /// Max height for the associated video stream
+        /// </summary>
+        public int maxHeight { get; set; }
+        /// <summary>
+        /// Numerator of framerate for the associated video stream
+        /// </summary>
+        public int num { get; set; }
+        /// <summary>
+        /// Denominator of framerate for the associated video stream
+        /// </summary>
+        public int den { get; set; }

```

Comment:
Why not use usually used formatting like a `60`, `24`, `120`?

``` c#
public struct ESFrameRate 
{
    public int Numerator {get;set;}
    public int Denominator {get;set;}
}
```
```suggestion
        public ESFrameRate FrameRate { get; set; }

```

Reply:
This is good idea. Thank you.